src/views/partials/header.hbs

```diff
@@ -1,5 +1,13 @@
     <div class="header">
-        <div class="left">LOGO</div>
-        <div class="right">list</div>
+        <div class="logo">voyage</div>
+        <div class="menu fas fa-bars">
+            <ul class="main-menu">
+                <a href="#" class="minu-link"><li class="menu-item"><span class="fas fa-question"></span> about us</li></a>
```

Comment:
is this a spelling error and the class is supposed to be menu-link?
Names should be as descriptive as possible and that includes being spelled correctly.
Aside from that the pull request will be ready to merge.